src/main/menus/tray.js

```diff
@@ -29,6 +29,14 @@ function createTemplate(mainWindow, config) {
     }), {
       type: 'separator'
     }, {
+      label: 'Settings',
+      click: () => {
+        mainWindow.loadURL('file://' + __dirname + '/browser/settings.html');
+        mainWindow.show();
```

Comment:
This would not work properly when the window is minimized from the maximized state (#249). So you should handle the window like [other shortcuts](https://github.com/mattermost/desktop/blob/f85337db6598e0a8fa2cd94c7868ce1b440314cc/src/main/menus/tray.js#L15-L26).